dataset/dictionary/quests.json5

```
@@ -2421,6 +2421,40 @@
     tags: [ "sumeru", "quest-story" ],
   },
 
+  //
+  // Tribal Chronicles
+  //
+  {
+    en: "Tribal Chronicles",
+    ja: "部族見聞",
+    zhCN: "部族纪闻",
+    pronunciationJa: "ぶぞくけんぶん",
+    tags: [ "quest-tribal" ],
```

I guess the Tribal Chronicles are Natlan-specific and will not be available in Snezhnaya and the other countries.
Therefore, how about add `natlan` tag to Tribal Chronicles itself?

```suggestion
    tags: [ "natlan", "quest-tribal" ],
```